MAINTAINING.md

```diff
@@ -36,12 +36,14 @@ the overall validity of the PR.
     Is the problem that the PR intends to solve clearly stated?
 2.  Is the solution presented in the PR as simple as possible to satisfy the
     requirements, but no simpler?
-3.  Is the PR functionally atomic? Does it address a single issue or should
-    it be split into multiple individual PRs?
+3.  Is the PR a manageable size for you to handle? As a general rule of thumb,
+    if the PR is likely to take more than a full day for one person to review,
+    then it may need to be broken down, at your discretion.
 4.  Is the solution well designed on a high level?
 5.  Do the concepts used by the PR make sense?
 6.  Does the PR break with existing concepts?
-7.  Is the structure of the PR itself valid?
+7.  Is there a clean commit history in the pulled branch? The commit history
```

Comment:
Please rephrase the second sentence. The history is not about "the work that has been done", but about grouping the actual diff.

Reply:
What should the diff show/how should it be grouped?

Reply:
@kaspar030 @danpetry can you come to a conclusion here?

Reply:
I've pushed a change to this... I.e. I rephrased as asked. @miri64 could you check the below for correctness pls?

Reply:
I'm okay with the current version, but since @kaspar030 requested the change I would give him a chance to react as well ;-).

Reply:
Ok, I think he will be ok with us proceeding, since no reply... would this then be ready to ACK?